cluster-autoscaler/cloudprovider/azure/README.md

```diff
@@ -25,6 +25,12 @@ az ad sp create-for-rbac --role="Contributor" --scopes="/subscriptions/<subscrip
 
 This will create a new [service principal][] with "Contributor" role scoped to your subscription. Save the JSON output, because it will be needed to configure the cluster autoscaler deployment in the next step.
 
+## Scaling a node group to and from 0
```

Comment:
from 0 for VMSS? since this only works for VMSS

Reply:
Updated